src/Utils/axiosRetryHandler.ts

```diff
@@ -83,4 +88,13 @@ export function isRetryableError(error: AxiosError): boolean {
       (error.response.status >= 500 && error.response.status <= 599))
   );
 }
-export default axiosRetryHandler;
\ No newline at end of file
+
+export function axiosRetryHandlerWithNotFound(axios: AxiosInstance, axiosRetryOptions: IAxiosRetryOptions): void {
```

Comment:
Can we combine that two AxiosRetry handlers into one? I think it'll be confusing having two of them, doing almost the same thing, especially since they both accept retry options.
If we need to add another condition in the future it will get really messy.

Reply:
its a wrapper around the existing shouldRetry logic